src/Operations/CompareMeshes.cc

```diff
@@ -109,6 +110,26 @@ bool CompareMeshes(Drover &DICOM_data,
         }
         second_max_distance = std::max(min_distance, second_max_distance);
     }
+
+    double sumx = 0, sumy = 0, sumz = 0;
+    for (auto & vertex : mesh1->meshes.vertices) {
+        sumx += vertex.x;
+        sumy += vertex.y;
+        sumz += vertex.z;
+    }
+
+    vec3 centroid1 = vec3(sumx/size(mesh1->meshes.vertices) , sumy/size(mesh1->meshes.vertices) , sumz/size(mesh1->meshes.vertices));
+
+    sumx =0, sumy=0 , sumz = 0;
+    for (auto & vertex : mesh2->meshes.vertices) {
+        sumx += vertex.x;
+        sumy += vertex.y;
+        sumz += vertex.z;
+    }
+
+    vec3 centroid2 = vec3(sumx/size(mesh1->meshes.vertices) , sumy/size(mesh1->meshes.vertices) , sumz/size(mesh1->meshes.vertices));
```

Comment:
I think we talked about this, but this really isn't the best way to find the centroid since if the vertices aren't distributed evenly it will give a completely wrong answer. In `SimpleMeshing.cc` this is found for individual contours as `contour.Average_Point()`, but looking at its implementation it seems it does the same thing (with a comment acknowledging that its pretty bad). 

For just evaluation metrics is probably fine though.